pyproject.toml

```diff
@@ -5,7 +5,7 @@ build-backend = "poetry.masonry.api"
 [tool.poetry]
 # Description
 name = "noos-inv"
-version = "0.0.11"
+version = "0.0.12"
 description = "Shared workflows across CI/CD pipelines"
 # Credentials
 license = "MIT"
```

Comment:
🐼 

A version left behing in the "classifiers" section of the `pyproject.toml`